client/main.lua

```diff
@@ -343,14 +346,8 @@ CreateThread(function()
             if cache.vehicle and not isHotwiring then
                 sleep = 1000
                 local plate = qbx.getVehiclePlate(cache.vehicle)
-
-                if GetPedInVehicleSeat(cache.vehicle, -1) == cache.ped
-                    and not hasKeys(plate)
-                    and not isBlacklistedVehicle(cache.vehicle)
-                    and not areKeysJobShared(cache.vehicle)
-                then
+                if cache.seat == -1 and not hasKeys(plate) and not isBlacklistedVehicle(cache.vehicle) and not areKeysJobShared(cache.vehicle) then
```

Comment:
This is another style decision team needs to align on. I personally prefer the former.

Reply:
I personally don't like the difference in indentation, mostly not due to the first condition being at the if statement whereas the rest are under it. I'd rather scroll to the right and focus on it than see it like this